driver/docker/docker_test.go

```diff
@@ -38,14 +42,19 @@ func TestDriver_GetConfigurationOptions(t *testing.T) {
 		err := d.applyConfigurationOptions()
 		is.Nil(err)
 
-		expectedContainerCfg := &container.Config{
+		expectedContainerCfg := container.Config{
 			User: "cnabby",
 		}
-		expectedHostCfg := &container.HostConfig{
+		expectedHostCfg := container.HostConfig{
 			Privileged: true,
 		}
-		cfg, hostCfg := d.GetContainerConfig()
+
+		cfg, err := d.GetContainerConfig()
+		is.Nil(err)
 		is.Equal(expectedContainerCfg, cfg)
+
+		hostCfg, err := d.GetContainerHostConfig()
+		is.Nil(err)
```

Comment:
This is feedback on when to use `Nil(error)` vs `NoError(error)`. The first will give you:

```
Expected nil, but got: &errors.errorString{s:"carolyn was here"}
```

The second gives you:

```
Received unexpected error:
carolyn was here
```

I'm not asking that you change anything, just sharing why in the future you may want to pick `NoError` instead for the improved failure message.

Reply:
`is.TooValuableToIgnore(feeback) = true` 😄